xdocs/changes.xml

```diff
@@ -208,12 +208,18 @@ Summary
 <p>We thank all contributors mentioned in bug and improvement sections above:
 </p>
 <ul>
+  <li>Arnout Engelen (raboof at github)</li>
+  <li>Clay Johnson (clayburn at github)</li>
+  <li>David Getzlaff (dgetzlaf at github)</li>
```

Comment:
I wonder if we would like to use real links to GitHub profiles instead of `... at github`.
Any thoughts?

Reply:
Why not.